### DISCUSSION TITLE
flet pack can not import name decode from pyinstaller

### DISCUSSION THREAD

**Discussion by m0555** [flet-dev/flet#1065] [Q&A] [answered]
`flet pack myapp.py` gives me following error (Windows 10):  
`Please install PyInstaller module to use flet pack command: cannot import name 'decode' from 'PyInstaller.utils.win32.versioninfo' (C:\Users\myusername\AppData\Local\Programs\Python\Python311\Lib\site-packages\PyInstaller\utils\win32\versioninfo.py)`

Same problem if I try to pack a demo app from the flet website

But no problem deploying a webapp like `flet publish myapp.py`, so I assume that the pyinstaller path is found and works.

Thanks for any help!

**Comment by thetrotfreak** [comment, marked as answer]
@m0555 , you should read the issue #1064 .

- It's been **fixed** in the commit 85043d25eaa7531bd069ec8cfa7b4389b47c740d, though not a part of the **flet release** `0.4.1`

- Use the pre-release version instead.
`pip install flet --upgrade --pre`

- Also, you'd need to manually install `PyInstaller` through pip as it's a **dep**.
`pip install pyinstaller`

### Note:
- If you want to stick with `flet v0.4.1`, then use `PyInstaller v5.7.0`

**Reply by m0555** [reply]
it works, thanks a lot!
